lib/rails_graylog/logger.rb

```diff
@@ -66,7 +66,9 @@ def normalize_message(severity, message)
     end
 
     def generate_short_message(message, writing_object = nil)
-      writing_object.nil? ? "#{message.to_s[0...25]}..." : "#{writing_object.class.name}_#{writing_object.id}"
+      return message[:short_message] if message.is_a?(Hash) && message.key?(:short_message)
+      m = message.is_a?(Hash) && message.key?(:message) ? message[:message] : message
+      writing_object.nil? ? "#{m.to_s[0...25]}..." : "#{writing_object.class.name}_#{writing_object.id}"
```

Comment:
shouldn't we add the `...` only if message is longer than 25 characters?

Reply:
Valid point, didn't want to disrupt the current mutations it's doing. I'll take a peek.